test/lib/specHelperSpec.js

```diff
@@ -108,5 +115,17 @@ describe('Fluxxed up test helpers', function() {
       // Signal that test should be finished:
       rig.finish()
     })
+
+    it('toggles the radio button', () => {
+      rig.toggleRadioButton('.radio-input')
+
+      expect(rig.domNode.find('.radio-input')[0].checked).to.equal(true)
```

Comment:
we need to use the whole shebang here. The test should use the whole form like the others do, so we can know, reliably, that the React inner state got updated.